### PR TITLE
Fix periodic ci-op config mounts

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @petr-muller

### CODE REVIEW

File: ci-operator/jobs/openshift/knative-build/openshift-knative-build-periodics.yaml [hunk 1]
```diff
@@ -76,7 +76,7 @@ periodics:
         valueFrom:
           configMapKeyRef:
             key: openshift-knative-build-release-0.2.yaml
-            name: ci-operator-misc-configs
+            name: ci-operator-master-configs
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/knative-build/openshift-knative-build-periodics.yaml]: Aaaand these should be in `-misc-`, I think :)

https://github.com/openshift/release/blob/master/cluster/ci/config/prow/plugins.yaml#L627

File: ci-operator/jobs/openshift/knative-eventing-sources/openshift-knative-eventing-sources-periodics.yaml [hunk 2]
```diff
@@ -76,7 +76,7 @@ periodics:
         valueFrom:
           configMapKeyRef:
             key: openshift-knative-eventing-sources-release-0.2.yaml
-            name: ci-operator-misc-configs
+            name: ci-operator-master-configs
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/knative-eventing-sources/openshift-knative-eventing-sources-periodics.yaml]: misc

File: ci-operator/jobs/openshift/knative-eventing/openshift-knative-eventing-periodics.yaml [hunk 3]
```diff
@@ -76,7 +76,7 @@ periodics:
         valueFrom:
           configMapKeyRef:
             key: openshift-knative-eventing-release-0.2.yaml
-            name: ci-operator-misc-configs
+            name: ci-operator-master-configs
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/knative-eventing/openshift-knative-eventing-periodics.yaml]: `misc`

File: ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-periodics.yaml [hunk 4]
```diff
@@ -76,7 +76,7 @@ periodics:
         valueFrom:
           configMapKeyRef:
             key: openshift-knative-serving-release-0.2.yaml
-            name: ci-operator-misc-configs
+            name: ci-operator-master-configs
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-periodics.yaml]: `misc`

File: ci-operator/jobs/openshift/knative-build/openshift-knative-build-periodics.yaml (hunk 1 above)
Comment (stevekuznetsov) [reply ci-operator/jobs/openshift/knative-build/openshift-knative-build-periodics.yaml]: my `sed` fu is so bad

**Pull request merged**